Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN yum install -y python-requests && \
     yum install -y openstack-ironic-api openstack-ironic-conductor crudini \
         iproute iptables dnsmasq httpd qemu-img-ev iscsi-initiator-utils parted gdisk ipxe-bootimgs psmisc sysvinit-tools \
         mariadb-server python2-chardet && \
-    yum clean all
+    yum clean all && rm -rf /var/cache/yum/*
```

Comment:
Do we also need to purge `/var/cache/dnf` for newer operating systems?

Reply:
That's a good point, thanks! On an UBI8 container, looks like ~300k of data remains in `/var/cache/dnf` after `dnf clean all`